x/profiles/types/models_chain_links_test.go

```diff
@@ -920,7 +1008,7 @@ func TestHexAddress_Validate(t *testing.T) {
 		},
 		{
 			name:      "address value shorter than prefix returns error",
-			address:   types.NewHexAddress("0", "0x"),
+			address:   types.NewHexAddress("0x", "0x"),
```

Comment:
Using same length tests the boundary condition of this:
https://github.com/desmos-labs/desmos/blob/b20b8c895232a9372a45c58395df0cb808998a39/x/profiles/types/models_chain_links.go#L603C5-L603C34